docs/app/pages/css/css-sections/icons/ux-icons/icon-preview/icon-preview.component.html

```diff
@@ -0,0 +1,64 @@
+<button uxFocusIndicator
+        tabindex="0"
+        class="icon-demo btn btn-link button-secondary btn-text"
+        [uxPopover]="iconCodeSnippet"
```

Comment:
When I try to select the copy button the popover jumps (this may be related to the issue above):

[![Image from Gyazo](https://i.gyazo.com/ffb1d169bcf2f5587e8ff0d06ef51299.gif)](https://gyazo.com/ffb1d169bcf2f5587e8ff0d06ef51299)

Reply:
Fixed with the above fix